internal/handlers/sqlite/msg_aggregate.go

```diff
@@ -400,21 +400,26 @@ func processStagesStats(ctx context.Context, closer *iterator.MultiCloser, p *st
 			avgObjSize = collStats.SizeCollection / collStats.CountObjects
 		}
 
+		indexSizes := types.MakeDocument(len(collStats.IndexSizes))
+		for _, indexSize := range collStats.IndexSizes {
+			indexSizes.Set(indexSize.Name, indexSize.Size)
+		}
+
 		doc.Set(
 			"storageStats", must.NotFail(types.NewDocument(
-				"size", collStats.SizeTotal,
+				"size", int32(collStats.SizeTotal),
```

Comment:
```
	// MongoDB uses "numbers" that could be int32 or int64,
	// FerretDB always returns int64 for simplicity.
```

I wonder if that's also the case there and we see `int32` only because we don't create large collections

Reply:
That's good point.
MongoDB returns sizes in int32 for all test cases we have, but I haven't checked what happens if the size is very large. Let me check.

Reply:
Yeah it was exactly the case https://github.com/FerretDB/FerretDB/pull/3575/commits/dcb095d383eb4b47caa71d4fdfd8c9c3d9b46ec6

So using `int32` is unwise.

Reply:
Let's keep using `int64` only for now. It makes test a bit messy.